src/app/services/roles.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+
+enum appRoles { anonymous = 0, admin = 1, user = 2, view = 3, data_view = 4 };
```

Comment:
we can have this object from API call?

Reply:
It would be better to show Role name in GUI as:

1. Administrator | Admin 
2. Editor | Edit 
3. Viewer | View
4. Data Viewer | Data View

Reply:
>we can have this object from API call?
If we will do that, either we need to fetch all roles every time in this service and map, that to me is not a good idea as  we guards the routes based on role and in role guards we are accessing this service to know the role,
Else we need to hold all the roles in the session that we are fetching on login? 


* To change the naming on UI better to do it using new JIRA.

Reply:
OK, please file a Jira and add notes which dropdown need binding to these value as options.

Reply:
I wonder if needed the API call then which we requested for data view, if we have to hard-code these. Obviously anonymous = 0 needs to be augmented.